frontend/src/components/spinner/index.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
```

Comment:
What I meant was that you can remove this file and use Icon component isntead. The button also accepts a type "icon" i think, that you can insert the loading icon in